SharedCode/Repository/DB/PokemonRepositoryLocal.cs

```diff
@@ -118,6 +117,54 @@ public async Task<Result<List<PokemonLocal>>> GetPokemonLocalListAsync()
             }
             return Result.Fail<List<PokemonLocal>>("Can't get info from db");
         }
+
+        public void StoreEvolutionChain(EvolutionChainResponse evolutionChain)
```

Comment:
Are you using this function, cause it looks like you are not calling it

Reply:
This function should be use when fetch the pokemon details.

Reply:
ok